dogen/plugins/cct.py

```diff
@@ -91,6 +91,14 @@ def prepare(self, cfg):
 
         os.makedirs(target_modules_dir)
 
+        local_modules_dir = os.path.join(os.path.dirname(self.descriptor),'cct')
+        self.log.debug('Found existing modules in directory: %s' % local_modules_dir)
```

Comment:
I'm logging real modules name if its get copied - this is IMHO ok as Debug its only telling some convention.